runtime-sdk/modules/evm/Cargo.toml

```diff
@@ -9,6 +9,7 @@ license = "Apache-2.0"
 [dependencies]
 cbor = { version = "0.1.7", package = "oasis-cbor" }
 oasis-runtime-sdk = { path = "../.." }
+oasis-core-runtime = { git = "https://github.com/oasisprotocol/oasis-core" }
```

Comment:
This should not be needed as core is available as `oasis_runtime_sdk::core`.

Reply:
Why is this back now?

Reply:
I'm not familiar with these specifications. I'm very sorry for the extra workload. Thank you very much for your guidance